src/ui/public/courier/__tests__/saved_object.js

```diff
@@ -0,0 +1,213 @@
+/**
+ * Tests functionality in ui/public/courier/saved_object/saved_object.js
+ */
+
+import ngMock from 'ng_mock';
+import expect from 'expect.js';
+import sinon from 'auto-release-sinon';
+
+import BluebirdPromise from 'bluebird';
+import SavedObjectFactory from '../saved_object/saved_object';
+import { stubMapper } from 'test_utils/stub_mapper';
+import IndexPatternFactory from 'ui/index_patterns/_index_pattern';
+
+describe('Saved Object', function () {
```

Comment:
I don't know if there is a rule for this, but I think I prefer not specifying the path, otherwise you'd have to update the test if you move the file and I could see that easily being forgotten.